packages/create-catalyst/src/utils/clone-catalyst.ts

```diff
@@ -103,42 +78,29 @@ export const cloneCatalyst = async ({
 
   writeJsonSync(join(projectDir, 'package.json'), packageJson, { spaces: 2 });
 
-  const tsConfigJson = z
-    .object({
-      compilerOptions: z
-        .object({
-          declaration: z.boolean().optional(),
-          declarationMap: z.boolean().optional(),
-          paths: z
-            .object({
-              '@bigcommerce/components/*': z.string().array().optional(),
-            })
-            .passthrough(),
-        })
-        .passthrough(),
-      include: z.array(z.string()).optional(),
-    })
-    .passthrough()
-    .parse(
-      merge(
-        {},
-        readJsonSync(join(projectDir, 'tmp/tsconfig.json')),
-        readJsonSync(join(projectDir, 'tsconfig.json')),
-      ),
-    );
-
-  delete tsConfigJson.compilerOptions.declaration;
-  delete tsConfigJson.compilerOptions.declarationMap;
-
-  tsConfigJson.compilerOptions.paths['@bigcommerce/components/*'] = ['./components/ui/*'];
-
   if (!includeFunctionalTests) {
+    const tsConfigJson = z
+      .object({
+        compilerOptions: z
+          .object({
+            declaration: z.boolean().optional(),
+            declarationMap: z.boolean().optional(),
+          })
+          .passthrough(),
```

Comment:
I think we should be able to remove this bit now (and the `delete` statements below).